exporter/tanzuobservabilityexporter/README.md

```diff
@@ -46,14 +53,25 @@ exporters:
 service:
   pipelines:
     traces:
-      receivers: [examplereceiver]
-      processors: [batch]
-      exporters: [tanzuobservability]
+      receivers: [ examplereceiver ]
+      processors: [ batch ]
+      exporters: [ tanzuobservability ]
 ```
 
 ### Advanced Configuration
 
-This exporter uses [queuing and retry helpers](https://github.com/open-telemetry/opentelemetry-collector/blob/main/exporter/exporterhelper/README.md) provided by the core OpenTelemetry Collector. The `retry_on_failure` and `sending_queue` features are enabled by default, but can be disabled using the options below.
+#### Processor
```

Comment:
Can we pluralize this to "Processors" since there may be multiple Collector Processors that we recommend. E.g. we already have `batch` as a default in our example config.

Reply:
Will make a change.